examples/generate_image_dataset/generate_images.py

```diff
@@ -100,8 +100,8 @@ def single_step(self, action=None, get_observation=False):
 
         obs, _, _, _ = env.step(
             action={
-                "set_location": np.array([pose["location_x"], pose["location_y"], pose["location_z"]], np.float64),
-                "set_rotation": np.array([pose["rotation_pitch"], pose["rotation_yaw"], pose["rotation_roll"]], np.float64)})
+                "set_location": np.array([pose["location_x_cms"], pose["location_y_cms"], pose["location_z_cms"]], np.float64),
```

Comment:
I don't like this change. I think I have advocated for including units in interface names previously, so I realize that I'm going back and forth here. And I agree with the general principle of being specific with the names of things and making things clear and unambiguous for the reader.

However, I think an even cleaner convention is to simply pass values directly through to Unreal unless noted otherwise, and label our interfaces without units. In other words, I think it's better to just say `location_x`. If the reader is wondering what the units are, they can just remember our simple and consistent rule: _SPEAR always passes values directly through to Unreal unless noted otherwise, so units are whatever Unreal expects_.

I also think abbreviating centimeters as `cms` is confusing because it could be interpreted as `centimeters` or `centimeter*seconds`. The correct interpretation is pretty obvious in this case, but for other actions, scaling constants, etc throughout our code, the correct interpretation wouldn't always be so clear.

Also, if we labeled this column as `location_x_cms`, then I'd want to label tons of other things in our codebase with the correct units (e.g., dozens of config parameters, etc). That's too much effort and I'd rather be consistent. (There are a couple of parameters in `SPEAR` that are labeled with units, but those values aren't passed through to Unreal.)

Finally, as a matter of correctness, I don't think Unreal units are always centimeters. I think they're just centimeters by default. So the name `location_x_cms` might be misleading unless we're explicitly checking the `WorldToMetersScale` variable in Unreal.

Reply:
That makes sense. I've updated the code accordingly.